Lesson-7/style.css

```diff
@@ -0,0 +1,1071 @@
+    font-style: normal;
+    font-weight: 500;
+    font-size: 50px;
+    line-height: 59px;
+    color: #FFFFFF;
+    }
+    label {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #FFFFFF;
+    height: 21px;
+    }
+    .label {
+    margin-top: 100px;
+    display: flex;
+    margin-bottom: 24px;
+    }
+    #search {
+    white-space: nowrap;
+    }
+    #date {
+    margin-left: 230px;
+    white-space: nowrap;}
+
+    .place_to_request {
+    display: flex;
+    align-items: stretch;
+    height: 64px;
+    max-width: 1232px;
+    background: #ffffff;
+    border: 3px solid #f3f3f4;
+    border-radius: 8px;
+    margin-bottom: 120px;
+    }
+    .calendar{
+    width: 259px;
+    border: 3px solid #f4f4f3;
+    box-sizing: border-box;
+    border-radius: 8px;
+    }
+    input[type="submit"] {
+    width: 192px;
+    background: #3077C6;
+    border-radius: 8px;
+    font-family: Roboto;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    height: 64px;
+    }
+    input[type="submit"]::-moz-placeholder {
+    padding: 20px 57px;   
+    font-family: Roboto;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    }
+    input[type="submit"]::-webkit-input-placeholder { 
+    font-family: Roboto;
+    padding: 20px 57px;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    }
+    input[type="submit"]:-ms-input-placeholder { 
+    font-family: Roboto;
+    padding: 20px 57px;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    }
+    input[type="submit"]::-ms-input-placeholder { 
+    font-family: Roboto;
+    padding: 20px 57px;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    }
+    input[type="submit"]::placeholder { 
+    font-family: Roboto;
+    padding: 20px 57px;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF; 
+    } 
+    input[type="text"] {
+    width: 314px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #383838;
+    }
+    input[type="date"] {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    }
+    input[type="search"]
+    {
+    width: 458px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    color: #383838;
+    }
+    .phone_app {
+    display: flex;
+    justify-content: center;
+    margin-top: 120px;
+    }
+    .icon_google {
+    width: 135px;
+    height: 40px;
+    }
+    .icon_app_store {
+    width: 117px;
+    height: 38px;
+    }
```

Comment:
these links are in the wrong section. Also wrap images in links

Reply:
![image](https://user-images.githubusercontent.com/20189313/129491624-3d24acd3-8bff-41f6-91d1-345b1d4b6107.png)